### ISSUE TITLE
Clock icon (recalculate route with current departure time and weather) does not belong on Main panel

### ISSUE THREAD

**Issue by EVGrokker** [aesculus/EVTO-App-Feedback#73]
The main panel is where global app features are managed. There are two trip-related buttons (after Getting Started is removed) - New Trip and Resume Trip, both of which lead to the Plan Trip panel, which is where this recalculate function should live.

I suggest adding it to the Plan Trip menu 'Recalculate with current conditions', using the clock icon.

This would affect issue #61 which restructures the Plan Trip menu. I would suggest it be placed after the 'Calculate' menu item.

**Comment by aesculus** [comment]
Done. But the text seems too long for a menu. 
![image](https://cloud.githubusercontent.com/assets/283364/24369024/82f16242-12d7-11e7-8121-d5eacef77bc1.png)

**Comment by aesculus** [comment]
What about just leaving it as Start Trip?

**Comment by EVGrokker** [comment]
A couple of options.
1) Put the clock icon on the Plan Trip map. (Not a fan, rather keep the map uncluttered. But it allows easier access.)
2) Alternative text:
'Start trip'. Suggests that navigation will commence, which isn't really accurate.
'Update with current weather': Also too wordy
'Refresh weather' or 'Refresh weather conditions': Better, resetting time to now is implied.
'Refresh trip': Vague, but Help file could explain more.

I think I'd go with 'Refresh trip'.

**Comment by aesculus** [comment]
How about **Initialize Weather/Time**? That can fit OK

**Comment by EVGrokker** [comment]
I don't like 'Initialize' as much. Initialize suggests that we're somehow starting from scratch.  And I don't think you need to explicitly say 'Time', because getting a current weather forecast implies the segment is now.

I'm guessing that this same action would refresh the weather icons on the map when you add that future feature. I think the goal is to communicate "Update this segment with the current weather conditions along my route, assuming I'm ready to leave now".

Here's a couple more verbs that are in the right ball park:
Reload weather
Update weather
Waste Chris's server fees (haha)

Or, now that you've removed Calculate from the feature vocabulary, we could use:
Recalculate weather

I still like Refresh Trip.

**Comment by EVGrokker** [comment]
And now 'Refresh Trip' has been upgraded to 'Refresh Segment', my vote is for 'Refresh Segment'.

**Comment by aesculus** [comment]
OK. Refresh it will be. The help will explain it
